hw/ip/spi_host/spi_host.core

```diff
@@ -2,7 +2,7 @@ CAPI=2:
 # Copyright lowRISC contributors.
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
-name: "lowrisc:ip:spi_host:0.1"
+name: "lowrisc:ip:spi_host:1.0"
```

Comment:
i'm not super clear on the convention here, but i think the "1.0" here is technically supposed to correspond to an entry in `spi_host.prj.hjson`.  If that's the case, you might want to add a 1.0 entry there and indicate it is D2.  Kind of like [here](https://github.com/lowRISC/opentitan/blob/master/hw/ip/hmac/data/hmac.prj.hjson#L13).

Not a big deal in my opinion though.

Reply:
looks good.